infrastructure/general/setup-secure-wss.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Setup Secure WebSocket
+description: Instructions on setting up a secure socket for remote connections.
+---
+
+<!-- TODO: link relevant guides (bootnode, rpc) -->
+
+## Secure a WS Port
+
+A non-secure ws port can be converted to a secure WSS port by placing it behind an SSL-enabled proxy. This can be used to secure a bootnode or secure a RPC server. The SSL-enabled apache2/nginx/other proxy server redirects requests to the internal WebSocket and converts it to a secure (WSS) connection. For this, you will need an SSL certificate for which you can use a service like LetsEncrypt or self-signing.
+
+### Obtaining an SSL Certificate
+
+One easy way to get a free SSL certificate can be achieved by following the LetsEncrypt instructions
+([nginx](https://certbot.eff.org/instructions?ws=nginx&os=ubuntufocal){target=_blank}/[apache](https://certbot.eff.org/instructions?ws=apache&os=ubuntufocal){target=_blank}).
+This will auto-generate an SSL certificate and include it in your configuration.
+
+Alternatively, you can generate a self-signed certificate and rely on the raw IP address of your
+node when connecting to it. This is not preferable since you will have to whitelist the certificate
```

Comment:
```suggestion
node when connecting to it. This isn't preferable since you will have to allowlist the certificate
```
We do prefer the use of contractions for 'not' combinations. Readers are more likely to see the 'not' indicator if it is written "don't" or "won't" rather than as two words they might miss one if skimming, etc.